src/appshell/appshell.cpp

```diff
@@ -138,14 +141,18 @@ int AppShell::run(int argc, char** argv)
         // Setup Qml Engine
         // ====================================================
         QQmlApplicationEngine* engine = new QQmlApplicationEngine();
-        //! NOTE Move ownership to UiEngine
-        ui::UiEngine::instance()->moveQQmlEngine(engine);
 
-#ifndef Q_OS_WASM
-        const QString mainQmlFile = "/Main.qml";
-#else
+#ifdef KDAB_DOCKWIDGETS
+        KDDockWidgets::Config::self().setQmlEngine(engine);
+        const QString mainQmlFile = "/Main.KDAB.qml";
+#elif Q_OS_WASM
         const QString mainQmlFile = "/main.wasm.qml";
```

Comment:
not related to this task, but will be best rename to `Main.wasm.qml`

Reply:
renamed